persistence/src/commonMain/db_global/migrations/3.sqm

```diff
@@ -0,0 +1,4 @@
+import kotlin.Boolean;
+
+ALTER TABLE Accounts
+ADD COLUMN isPersistentWebSocketEnabled INTEGER AS Boolean NOT NULL;
```

Comment:
Quick question we were asking ourselves with Vitor, about how to standarize where we store this configs, we also have `UserConfigStorage`. Do you see any advantages to have it here ? And should we move what we have on `UserConfigStorage` also 🤔

Reply:
We will discuss it on the collective, so I'll close it :D